### DISCUSSION TITLE
react-datetime field remains opens when tabbing through formik

### DISCUSSION THREAD

**Discussion by pmoneysign** [jaredpalmer/formik#3112] [Q&A]
Hi, 
I have a few date fields that use react-datetime. When I tab through the fields, the date dropdown remains open. Any ideas on why it may be happening?

**Comment by johnrom** [comment]
I assume you are passing a custom onBlur prop (the one Formik provides, hopefully) to react-datetime, and it should trigger its close on blur.  You may need to create an onblur which both calls the close for datetime and calls formik's onblur. This may not be a good place to get help with that library, but hopefully this helps you find a path forward.
